tslint.json

```diff
@@ -21,7 +21,7 @@
         ],
         "quotemark": [
             true,
-            "double"
+            "single"
```

Comment:
Of course, *pick any and be consistent* is correct, but my choice is double.
HTML argument doesn't apply to JSX, just the other way around:
´´´ts
div = <div className="hola" />
span = <div className={'hola'} />
´´´
It's weird to change quotes here when an attribute becomes calculated ("hola" -> 'ho' + 'la').

I'm not trying to convince, I'm hardly will quit using doubles but I'm interested in reasons people use to make the decision. Community is also moving towards no semicolons, and I'm not in that ship either.